### DISCUSSION TITLE
How manage **dislike** actions

### DISCUSSION THREAD

**Discussion by AzadGh95** [maize-tech/laravel-markable#29] [Q&A] [answered]
How can I manage **dislike** actions?
Imagine I have a comment and a user should like or dislike my comment.
In this package, there are five functions for **like**, but I don't know how to add dislikes statistics.
also thanks in advance.

**Comment by riccardodallavia** [comment, marked as answer]
Hi @AzadGh95!

Usually, the dislike function is treated as a separate reaction.
You could easily create your own Dislike mark model following the [documentation](https://github.com/maize-tech/laravel-markable#custom-mark-model)!
